packages/user/src/migrations/addRoles.ts

```diff
@@ -0,0 +1,16 @@
+import { sql } from "slonik";
+
+import { ROLE_ADMIN, ROLE_USER } from "../constants";
+
+import type { ApiConfig } from "@dzangolab/fastify-config";
+
+const users = (config: ApiConfig) => {
+  return sql.unsafe`
+    INSERT INTO "st__roles" ("role") VALUES 
+    (${ROLE_ADMIN}),
+    (${config.user.role || ROLE_USER})
+    ON CONFLICT DO NOTHING;
+  `;
```

Comment:
@dipendraupreti there might be many roles that need to be supported by the app, the roles that need to add into the database should be get from the config